src/Raven.Studio/typescript/commands/database/query/queryCommand.ts

```diff
@@ -21,12 +24,20 @@ class queryCommand extends commandBase {
                 explanations: results.Explanations,
                 timings: results.Timings,
                 includes: results.Includes }) as pagedResultExtended<document>;
+        
         return this.query(this.getUrl(), null, this.db, selector)
             .fail((response: JQueryXHR) => {
                 if (response.status === 404) {
                     this.reportError("Error querying index", "Index was not found", response.statusText)
                 } else {
-                    this.reportError("Error querying index", response.responseText, response.statusText)
+                    const responseText = response.responseText;
+                    let errorTitle = "Error querying index";
+
+                    if (responseText.includes(queryCommand.missingEndOfQuery)) {
+                        errorTitle = "Incorrect query command syntax";
+                    }
+                    
+                    this.reportError(errorTitle, response.responseText, response.statusText);
```

Comment:
please use variable you declared above: responseText

Reply:
right